packages/react/src/components/cookieConsentCore/cookieConsentCore.js

```diff
@@ -408,22 +406,68 @@ export class CookieConsentCore {
     } else {
       window.dispatchEvent(new CustomEvent(cookieEventType.CHANGE, { detail: { acceptedGroups } }));
       if (!this.#settingsPageElement) {
-        this.#announceSettingsSaved();
         this.removeBanner();
+        // removeBanner() removes the setTimeout that shows notification
```

Comment:
commented code?

Reply:
It's a two line comment to warn that the order of function calls is essential and should never be changed.